packages/appnroll-stylelint-config/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@appnroll/stylelint-config",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Appnroll/appnroll-libs/tree/master/packages/appnroll-stylelint-config"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint --ext .js"
+  },
+  "homepage": "https://github.com/Appnroll/appnroll-libs/tree/master/packages/appnroll-stylelint-config#readme",
+  "author": "appnroll",
+  "license": "MIT",
+  "devDependencies": {
```

Comment:
The package depends on them to work. Hence those deps are not "dev".

Reply:
86d38e34359ddcafc0fca21cc4d2a22373a4805a